buildenv/jenkins/common/build

```diff
@@ -259,7 +259,12 @@ def build_all() {
                 archive()
             }
         } finally {
-            cleanWs()
+            try {
+		cleanWs()
+	    } catch(e) {
+		echo e.toString()
```

Comment:
Might be more useful to print the entire stack trace rather than just the exception message